### PR TITLE
Refactor website to websiteEditor

### LINKED ISSUE #34: Rename the app "Website" to "WebsiteEditor"
As an extension to https://github.com/StarcounterPrefabs/Website/issues/15
what about renaming this app to something like "Blender"?

As it fact this is the app that lets you blend the output of many apps which is just hidden behind inaccessible URLs.

//cc @warpech

### PR DESCRIPTION
**Pull request by bouguila**
fix #34
renamed website -> websiteEditor
changed URI in handler /website/ -> /websiteEditor/
renamed Directory src/website/wwwroot/website -> src/wesiteEditor/wwwroot/websiteEditor

### CODE REVIEW

**Comment by warpech** [comment]
@un-tone could you pls review?

**Comment by warpech** [comment]
> 1. Please resolve conflicts

I assume @un-tone means merge the current `develop` to `website-refactor-#34`

> Also I have some questions:
>
> 1. How about slot names in *.html files? For example shouldn't website/global-style to be renamed to website-editor/global-style?
> 2. The same question about class names, e.g. website-table__control.
> 3. Does we really need to keep these file names:
wwwroot\WebsiteEditor\css\website.css
wwwroot\WebsiteEditor\images\website.png

In my opinion it is: 

1. good point, @bouguila please rename to `websiteeditor/global-style`, etc
2. good point, please rename to `websiteeditor-table__control`, etc
3. good point, please rename to `websiteeditor.css` and `websiteeditor.png`

**Comment by PatrykSzwer** [comment]
Ping @bouguila

**Comment by warpech** [comment]
There are still some classes that should be renamed, e.g. `website-sidebartemplate` -> `website-sidebarsurface`

File: src/WebsiteEditor/wwwroot/WebsiteEditor/surfaces/DefaultSurface.html [hunk 1]
```diff
@@ -3,16 +3,16 @@
         body {
             margin: 0;
         }
-        .website-defaulttemplate-topbar > juicy-composition,
-        .website-defaulttemplate-topbar > juicy-composition > starcounter-include > juicy-composition {
+        .website-defaultsurface-topbar > juicy-composition,
```
Comment (warpech) [inline src/WebsiteEditor/wwwroot/WebsiteEditor/surfaces/DefaultSurface.html]: @bouguila, I assume you did this change based on a comment that I have made by mistake and later deleted... Sorry.

This change was already done in https://github.com/StarcounterApps/Website/pull/119

Can you just please merge `develop` to `website-refactor-#34`?

Comment (bouguila) [inline src/WebsiteEditor/wwwroot/WebsiteEditor/surfaces/DefaultSurface.html]: yes, no problem!

**Comment by PatrykSzwer** [comment]
There are again some conflicting files. @bouguila please investigate.

**Comment by un-tone** [comment]
There are building errors. Please fix.

Also there are 2 classes defined in `Website.Api` namespace while all other classes are defined in `Website`. I would recommend to change it while you are here in order to follow common style of the project.

**Comment by un-tone** [comment]
@bouguila the common style in the project is to define classes in `Website` and `WebsiteProvider` namespaces. Sorry if I told unclear. I mean to change `Website.Api` namespace to `Website`

**Comment by bouguila** [comment]
@un-tone ok i think i totally misunderstood you, but since Website is now called WebsiteEditor, don't you think that it would be more convenient if the namespace is WebsiteEditor also?

**Comment by un-tone** [comment]
Sure, `WebsiteEditor.Api` namespace should be changed to `WebsiteEditor`. Sorry again :-). 

Also I noticed that there are unnecessary
files in the repo https://github.com/StarcounterApps/Website/tree/website-refactor-%2334/src/Website/wwwroot/Website/surfaces. I'm sure something was happened when merging with `develop` branch. Could you remove it and check that not any changes was missed in this files (`src\WebsiteEditor\wwwroot\WebsiteEditor\surfaces\` directory) during that merge?

**Comment by bouguila** [comment]
@un-tone, the files under https://github.com/StarcounterApps/Website/tree/website-refactor-%2334/src/Website/wwwroot/Website/surfaces are necessary due to the rename of templates -> surfaces

**Comment by un-tone** [comment]
@bouguila after your renaming those files are located in [src/WebsiteEditor/wwwroot/WebsiteEditor/surfaces](https://github.com/StarcounterApps/Website/tree/website-refactor-%2334/src/WebsiteEditor/wwwroot/WebsiteEditor/surfaces) (in `WebsiteEditor` directories), I see it.
So the whole [src/Website](https://github.com/StarcounterApps/Website/tree/website-refactor-%2334/src/Website) folder should be removed.

**Comment by bouguila** [comment]
@un-tone are we good to go now?

**Comment by bouguila** [comment]
@un-tone great, thank you!

**Pull request merged**